packages/server/src/standalone/index.ts

```diff
@@ -0,0 +1,79 @@
+import { json } from 'body-parser';
+import cors from 'cors';
+import express from 'express';
+import http from 'http';
+import type { AddressInfo, ListenOptions } from 'net';
+import { format as urlFormat } from 'url';
+import type { ApolloServer } from '../ApolloServer';
+import { ExpressContext, expressMiddleware } from '../express';
+import type { BaseContext, ContextFunction } from '../externalTypes';
+import type { WithRequired } from '../types';
+
+interface HTTPServerOptions<TContext extends BaseContext> {
+  context?: ContextFunction<[ExpressContext], TContext>;
+  graphqlPath?: string;
```

Comment:
Hmm. AS3's standalone server doesn't let you configure graphqlPath (it's always `/`). That kinda makes sense — the only thing the server does is serve GraphQL so what would be the point of putting it on a subpath (ie, the root won't do anything!).

Of course changing this will require editing the whole integration test suite to delete `/graphql` but that's probably reasonable anyway. Makes the tests simpler. You may ask how this worked in AS3 and the answer is we didn't integration-test the standalone server :)

Reply:
[d2149b8](https://github.com/apollographql/apollo-server/pull/6420/commits/d2149b84f271489474f056d47a759e8e22e28493)